components/prism-typoscript.js

```diff
@@ -0,0 +1,31 @@
+    }, {
+        pattern: /pagets__.*(?:\s|=|.)/g
+    }],
+    'function': [{
+        pattern: /@import/g,
+        greed: true,
+    }, {
+        pattern: /{[$#].*}|{(?:register|field|cObj):.*}|(?:TSFE|file):.*\n?\s?/g
+    }],
+    'string': [{
+        pattern: /^([^=]*=[< ]?)(?:(?!]\n).)*/g,
+        lookbehind: true,
+        inside: {
+            'function': {
+                pattern: /{[$#].*}|{(?:register|field|cObj):.*}|(?:TSFE|file):.*\n?\s?/g
+            },
+            'keyword': {
+                pattern: /\b(?:_GIFBUILDER|FLUIDTEMPLATE|CARRAY|CASE|CLEARGIF|COA|COA_INT|CONSTANTS|CONTENT|EDITPANEL|EFFECT|FILE|FORM|FRAME|FRAMESET|GIFBUILDER|global|globalString|globalVar|GMENU|GMENU_FOLDOUT|GMENU_LAYERS|GP|HMENU|HRULER|HTML|IENV|IMAGE|IMG_RESOURCE|IMGMENU|IMGMENUITEM|IMGTEXT|JSMENU|JSMENUITEM|LOAD_REGISTER|PAGE|RECORDS|RESTORE_REGISTER|TEMPLATE|TEXT|TMENU|TMENU_LAYERS|TMENUITEM|USER|USER_INT)\b/g
+            }
+        }
+    }, {
+        pattern: /\n?\s+["'].*["']/g,
```

Comment:
Why does this start with `\n?\s+`

Reply:
I've reworked this too, will add it asap.